genomes/management/lib/__init__.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import logging
+
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from analyses.models import Analysis
+
+logger = logging.getLogger(__name__)
+
+
+class EMGBaseCommand(BaseCommand):
```

Comment:
This is from the old days.. I'm not sure if this is the best way of composing cli commands. I like click a lot.. so maybe - https://github.com/django-commons/django-click to make it easier to build cli commands? @SandyRogers @MGS-sails

Reply:
Thanks @mberacochea I checked out djclick, seems pretty nice to have. But I think we don't need to install it this time, as we're no longer using these management commands for now. Just the Prefect flow